sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -91,6 +112,18 @@ export class AppConfigurationClient {
     }
   }
 
+  /**
+   * Add a setting into the Azure App Configuration service, failing if it 
+   * already exists.
+   * 
+   * Example usage:
+   * ```ts
+   * const result = await client.addConfigurationSetting("MyKey", { label: "MyLabel", value: "MyValue" });
+   * ```
+   * @param key The name of the key.
```

Comment:
Minor: The key is the id someone uses to identify a ConfigurationSetting, so maybe something along the lines of the below?
```suggestion
   * @param key The name of the ConfigurationSetting to be added.
```
Similar changes could be applied throughout.

Reply:
I actually had a longer name for it but poking through the documentation (and portal) they always just refer to it as a key-value pair or a key (ie, there's no name for it like ConfigurationSetting or even appsetting, for instance). 

So I'm tempted to leave it just so it's consistent with the public docs.

Reply:
Nope, if that matches the docs and the portal that works for me.